### PR TITLE
Search by location

### PR DESCRIPTION
**Pull request by marysalemme**
Please?
🐶

### CODE REVIEW

File: app/controllers/search.rb [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+    session[:location] = params[:location]
     redirect '/search/results'
   end
-  
+
   get '/search/results' do
-    @filtered_spaces = Space.all(:price.gt => session[:price_min], :price.lt => session[:price_max])
+    @filtered_spaces = Space.all(:price.gt => session[:price_min],
+                                 :price.lt => session[:price_max],
+                                 :location => session[:location])
```
Comment (tbscanlon) [inline app/controllers/search.rb:13]: Much nicer :1st_place_medal:

**Review by tbscanlon — APPROVED** [review]
:shipit: :shipit: :shipit:

**Review by vhonebon1 — APPROVED** [review]
:poodle:

**Comment by marysalemme** [comment]
All good!!

**Pull request merged**